apps/bmd/src/utils/Card.ts

```diff
@@ -233,17 +233,17 @@ export class MemoryCard implements Card {
   /**
    * Inserts a simulated in-memory card with specified long and short values.
    */
-  public insertCard<T>(
+  public insertCard(
     shortValue?: string,
-    longValue?: T | string | Uint8Array
+    longValue?: string | Uint8Array
```

Comment:
Allowing any arbitrary value and `JSON.stringify`ing it encouraged bad habits and made it hard/impossible to match election hashes properly.